src/components/Dashboard/StaffNotation.vue

```diff
@@ -86,83 +86,78 @@
 
             <v-list-item
+              v-if="isFirm"
               data-type="record-conversion"
               @click="goToConversionFiling()"
-              :disabled="disabled || !isFirm || isAdminFreeze"
+              :disabled="!isAllowed(AllowableActions.RECORD_CONVERSION)"
             >
               <v-list-item-title>
                 <span class="app-blue">Record Conversion</span>
               </v-list-item-title>
             </v-list-item>
 
-            <template v-if="(isFirm || isCoop || isBenBcCccUlc) && !isHistorical">
-              <v-list-item
-                data-type="administrative-dissolution"
-                @click="showAdministrativeDissolutionDialog()"
-                :disabled="disabled || isAdminFreeze"
-              >
-                <v-list-item-title>
-                  <span class="app-blue">Administrative Dissolution</span>
-                </v-list-item-title>
-              </v-list-item>
-            </template>
-
-            <template v-if="isBenBcCccUlc && isHistorical">
-              <v-list-item
-                data-type="restoration"
-                @click="goToRestorationFiling(ApplicationTypes.CREATE_UI, FilingSubTypes.RESTORATION_FULL)"
-              >
-                <v-list-item-title>
-                  <span class="app-blue">Restore Company</span>
-                </v-list-item-title>
-              </v-list-item>
-            </template>
-
-            <template v-if="(isFirm || isCoop || isBenBcCccUlc) && isHistorical">
-              <v-list-item
-                data-type="put-back-on"
-                @click="showPutBackOnDialog()"
-              >
-                <v-list-item-title>
-                  <span class="app-blue">Put Back On</span>
-                </v-list-item-title>
-              </v-list-item>
-            </template>
-
-            <template v-if="!isHistorical">
-              <v-list-item
-                data-type="admin-freeze"
-                @click="showAdministerFreezeDialog()">
-                <v-list-item-title>
-                  <span class="app-blue">{{ isAdminFreeze ? 'Unfreeze Business' : 'Freeze Business' }}</span>
-                </v-list-item-title>
-              </v-list-item>
-            </template>
-
-            <template v-if="isBenBcCccUlc && !isHistorical">
-              <v-list-item
-                data-type="consent-continue-out"
-                @click="goToConsentContinuationOutFiling()"
-                :disabled="disabled || isAdminFreeze"
-              >
-                <v-list-item-title>
-                  <span class="app-blue">Consent to Continuation Out</span>
-                </v-list-item-title>
-              </v-list-item>
-            </template>
+            <v-list-item
+              v-if="!isHistorical"
+              data-type="administrative-dissolution"
+              @click="showAdministrativeDissolutionDialog()"
+              :disabled="!isAllowed(AllowableActions.ADMINISTRATIVE_DISSOLUTION)"
```

Comment:
My thought process here is that this is never applicable unless the businss is historical.